tests/extension/ai/converters.test.ts

```diff
@@ -0,0 +1,30 @@
+import * as assert from 'node:assert'
+
+// TODO(jeremy): Importing vscode here appears to cause this to fail when run as a unit-test using
+// npx runme run test:unit
+import * as vscode from 'vscode'
+import { describe, it } from 'vitest'
+import { GenerateCellsRequest } from '@buf/stateful_runme.community_timostamm-protobuf-ts/runme/ai/v1alpha1/ai_pb'
+import { CellKind } from '@buf/stateful_runme.community_timostamm-protobuf-ts/runme/parser/v1/parser_pb'
+
+import * as converters from '../../../src/extension/ai/converters'
+
+// TODO(jeremy): Add more tests assuming we do in fact need the converters.
```

Comment:
@sourishkrout I didn't flush out the unittests for the converters because I wasn't sure we actually needed them.  I wasn't quite sure how to structure the test if we do. It looks like importing vscode means we can no longer treat this as a unittest.

Reply:
👍  assuming you can reuse the Serializer's marshal and revive methods you should be good to forgo this.